lib/tic_tac_toe/board_protocols.ex

```diff
@@ -0,0 +1,7 @@
+defprotocol TicTacToe.BoardUpdate do
+  def place_token(board, position, token)
+end
+
+defprotocol TicTacToe.BoardInspect do
+  def get(board, position)
+end
```

Comment:
My own preference would probably to rename these 

* `TicTacToe.BoardUpdate` -> `TicTacToe.Board` - This would require renaming the existing `Board` to something more specific like `3By3Board` or whatever.
* `TicTacToe.BoardInspect` -> `TicTacToe.Board.Inspect`

No change needed, I just wanted to offer that up.

Reply:
Thanks